src/ORM/Table.php

```diff
@@ -460,17 +460,17 @@ public function schema($schema = null)
      * ### Example:
      *
      * ```
-     * protected function _initializeSchema(\Cake\Database\Schema\Table $table) {
+     * protected function _initializeSchema(\Cake\Database\Schema\TableSchema $table) {
      *  $table->columnType('preferences', 'json');
      *  return $table;
      * }
      * ```
      *
-     * @param \Cake\Database\Schema\Table $table The table definition fetched from database.
-     * @return \Cake\Database\Schema\Table the altered schema
+     * @param \Cake\Database\Schema\TableSchema $table The table definition fetched from database.
+     * @return \Cake\Database\Schema\TableSchema the altered schema
      * @api
      */
-    protected function _initializeSchema(Schema $table)
+    protected function _initializeSchema(TableSchema $table)
```

Comment:
I checked, and these typehint changes will cause strict errors for existing applications:

```
Strict (2048): Declaration of App\Model\Table\BookmarksTable::_initializeSchema() should be compatible with Cake\ORM\Table::_initializeSchema(Cake\Database\Schema\TableSchema $table) [APP/Model/Table/BookmarksTable.php, line 14]
```

I think reverting the type hint changes might be the best option if you still want to land this in `master`.